cli/src/cmd_net.rs

```diff
@@ -604,50 +893,50 @@ pub struct CmdBgpPeer {
 impl AuthenticatedCmd for CmdBgpPeer {
     async fn run(&self, client: &Client) -> Result<()> {
         match &self.subcmd {
-            BgpConfigPeerSubCommand::Add(cmd) => cmd.run(client).await,
-            BgpConfigPeerSubCommand::Del(cmd) => cmd.run(client).await,
+            BgpConfigPeerSubCommand::Set(cmd) => cmd.run(client).await,
+            BgpConfigPeerSubCommand::Delete(cmd) => cmd.run(client).await,
         }
     }
 }
 
 #[derive(Parser, Debug, Clone)]
 enum BgpConfigPeerSubCommand {
-    /// Add a BGP peer to a port configuration.
-    Add(CmdBgpPeerAdd),
+    /// Set a BGP peer on a port configuration.
+    Set(CmdBgpPeerSet),
 
     /// Remove a BGP from a port configuration.
-    Del(CmdBgpPeerDel),
+    Delete(CmdBgpPeerDel),
 }
 
-/// Add a BGP peer to a given switch port configuration.
+/// Set a BGP peer configuration.
 ///
 /// This performs a read-modify-write on the specified switch port
 /// configuration.
 #[derive(Parser, Debug, Clone)]
 #[command(verbatim_doc_comment)]
-#[command(name = "add")]
-pub struct CmdBgpPeerAdd {
-    /// Id of the rack to add the peer to.
+#[command(name = "set")]
+pub struct CmdBgpPeerSet {
+    /// Id of the rack to set the peer for.
```

Comment:
Might be more consistent to use "on", since that is what is used for `--switch` (line 923) and `--port` (line 927) below, or vice versa, as long as this `oxide system networking bgp peer set` is consistent with the inverse of `oxide system networking bgp peer delete`, and consistent with `oxide system networking bgp pref`, and consistent with `oxide system networking route set`, and consistent with the inverse of `oxide system networking route delete `.

Reply:
Done